test/LibraryManager.Test/Providers/FileSystem/FileSystemProviderTest.cs

```diff
@@ -254,44 +254,41 @@ public void Cleanup()
 
             ILibraryOperationResult result = await provider.InstallAsync(desiredState, CancellationToken.None);
             Assert.IsFalse(result.Success);
-            Assert.AreEqual("LIB005", result.Errors[0].Code);
+            Assert.AreEqual("LIB002", result.Errors[0].Code);
```

Comment:
We changed the order of  detection of errors with the new library validator.

Reply:
Can you change this test so that it is validating the appropriate error for this scenario?  It's no longer verifying a PathNotDefined error.

---
In reply to: [195449269](https://github.com/aspnet/LibraryManager/pull/164#discussion_r195449269) [](ancestors = 195449269)